src/todos/todos.controller.ts

```diff
@@ -1,4 +1,38 @@
-import { Controller } from '@nestjs/common';
+import { Body, Controller, Post, Get, Param, Delete, Put } from '@nestjs/common';
+import { CreateToDoDto } from './dto/create-todo.dto';
+import { TodosService } from './todos.service';
 
 @Controller('todos')
-export class TodosController {}
+export class TodosController {
+    
+    constructor(private todosService: TodosService) {
+
+    }
+
```

Comment:
Why did you declare the constructor if it is empty?

Reply:
To use service methods is controller.